.github/prompts/prompt.sh

```
@@ -0,0 +1,40 @@
+    "stop"     "Abort operation" \
+    "provide help"    "Enter prompt to help AI" \
+    3>&1 1>&2 2>&3)
+
+  case "$choice" in
+    continue)
+      echo "Continuing..."
+      ;;
+    stop)
+      echo "Stopping..."
+      ;;
+    provide help)
```

Shellcheck (0.11.0)

[error] Couldn't parse this case item. Fix to allow more checks.

(SC1073)